database/misc/couchdb_user

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2015, Maykel Moya <mmoya@mmoya.org>
+# Sponsored by ShuttleCloud http://shuttlecloud.com/
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: couchdb_user
+short_description: Add or remove CouchDB users.
+description:
+   - Add or remove CouchDB users from a remote host.
+version_added: "0.x"
```

Comment:
you can set it to "2.1" for now, if you work with me, we will be able to merge it :)